indico_toolkit/association/splitting.py

```diff
@@ -0,0 +1,32 @@
+import re
+from copy import deepcopy
+from itertools import zip_longest
+from typing import Any, Dict, Iterator
+
+
+def split_prediction(
+    prediction: Dict[str, Any], separator: str = r"\s*\n\s*"
```

Comment:
I'd define what the default separator is going to split on in the docstring

Reply:
Added to the docstring that the default behavior splits on newlines and also captures any adjacent whitespace.